src/main/java/bio/terra/cli/serialization/userfacing/resource/UFGcpNotebook.java

```diff
@@ -62,6 +66,10 @@ public void print(String prefix) {
     OUT.println(prefix + "Instance id:       " + instanceId);
     OUT.println(prefix + "Instance name: " + (instanceName == null ? "(undefined)" : instanceName));
     OUT.println(prefix + "State:         " + (state == null ? "(undefined)" : state));
+    OUT.println(prefix + "Metadata:");
+    for (Map.Entry<String, String> entry : metadata.entrySet()) {
```

Comment:
You can also do 
```
metadata.forEach((key, value) -> OUT.println("   " + key + ": " + value));
```
I think it's cleaner than peeling off the entrySet when you don't have to.

Reply:
great!